src/components/accounts/AccountFormAccountId.js

```diff
@@ -17,7 +17,7 @@ class AccountFormAccountId extends Component {
     }
 
     handleChangeAccountId = (e, { name, value }) => {
-        if (value.match(/[^a-zA-Z0-9_-]/)) {
+        if ((this.props.type !== 'send-money' && value.match(/[^a-zA-Z0-9_-]/)) || value.match(/[^a-zA-Z0-9._-]/)) {
```

Comment:
let's not hard code `send-money`. How about we pass a pattern in a prop and have a default value allowing subdomains?

I think the only place where we shouldn't allow them is account creation.

Reply:
Changes made. Send Money working fine, you can test it by sending it to `marcin.test`.